src/js/components/Pagination.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component, PropTypes } from 'react'
+
+const Pagination = (props) => {
+  const page = props.pagination
+  if(!page) return null
+  const firstPage = page._links.self === page._links.first
+  const lastPage = page._links.self === page._links.last
+
+  return (
+    <div className="PaginationControls">
+      <button
+        className={ firstPage ? 'usa-button-disabled' : '' }
+        onClick={ e => { if(!firstPage) props.getPreviousPage(page) }}
+      >Previous</button>
+      <div>Page <h4>{page._links.self.slice(-1)}</h4> of {Math.ceil(page.total/20)}</div>
+      <button
+        className={ lastPage ? 'usa-button-disabled' : '' }
+        onClick={ e => { if(!lastPage) props.getNextPage(page) }}
+      >Next</button>
+    </div>
+  )
+}
+
+export default Pagination
```

Comment:
Yeah, sorry 😞 . I meant PropTypes. I find it useful just to see what should be coming in.